ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -62,6 +62,21 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: ipi-aws
+- as: e2e-console
+  run_if_changed: frontend/
+  steps:
```

Comment:
@yapei do you need the cluster for testing? if yes, you should miss the workflow to create the cluster.
I guess it is workflow: ipi-gcp

Reply:
@kuiwang02  isn't `cluster_profile` the right place of specifying cluster?

Reply:
it is cluster_profile, but no create cluster. you could try it to see if your case execute

Reply:
`cluster_profile` just define the cluster type, but doesn't create the cluster.

Reply:
I see @kuiwang02 , updated